content/posts/android/Installing Kali Nethunter Lite on DivestOS 20.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Installing Kali Nethunter Lite on DivestOS 20"
+date: 2022-07-18
+tags: ['Operating Systems', 'Android', 'Security']
+author: Tommy
+---
+
+**Disclaimer**: This will turn your device into a purely penetration testing device, and it will be very insecure for any other purposes. Please do not do this on your daily driver.
```

Comment:
```suggestion
**This will turn your device into a purely penetration testing device, and it will be very insecure for any other purpose. Please do not do this on your daily driver.**

_This procedure should work without modification on many devices using the standard Android bootloader (not Samsung). However, every device has unique quirks that may require special attention._
```